sequencer/sequencesender.go

```diff
@@ -5,9 +5,9 @@ import (
 	"errors"
 	"fmt"
 	"math/big"
-	"strings"
 	"time"
 
+	ethman "github.com/0xPolygonHermez/zkevm-node/etherman"
```

Comment:
```suggestion
	"github.com/0xPolygonHermez/zkevm-node/etherman"
```

Reply:
this is not valid as it collides with the interface inside the package: https://github.com/0xPolygonHermez/zkevm-node/blob/develop/sequencer/interfaces.go#L32